File: src/app/class/blog.ts
export class Blog {
    id:number;
    title:string;
    image:string;
    description:string;
    type:string;

    constructor(props: any) {
        this.id = props.id;
        this.title = props.title;
        this.image = props.image;
        this.description = props.description;
        this.type = props.type;
    }
}

File: campaign_manager/gos/models/blog.py
from django.db import models

class Blog(models.Model):
    title = models.CharField(max_length=255)
    image = models.CharField(max_length=255)
    description = models.TextField()
    type = models.CharField(max_length=30, default="character")

    def __str__(self):
        return self.title